cg/test/management/subjects.api.spec.js

```diff
@@ -591,7 +591,7 @@ describe('List rectification requests', () => {
   });
 });
 
-describe('Rectification requests archive', () => {
```

Comment:
This was named like that on purpose, so the tests read like:
Rectification requests archive + Should list archived requests in pages bigger than 1 correctly

Now they read like:
Tests of listing archived rectification requests + Should list archived requests in pages bigger than 1 correctly

This should be more like natural sentences. The tests should only test, but the endpoints/class methods/functions should do whatever we're testing.

Reply:
It's more intuitive to have an overview of the test in the describe, but I see why it was named like that. I prefer to have a more general overview in the describe, since the it is read separately from the describe most of the time. I can rename it back, what do you think?